src/app_engine/probers.py

```diff
@@ -155,23 +187,23 @@ def create_collider_active_host(self, old_active_host, probing_results):
 
   def get(self):
     results = {}
-    for collider_instance in constants.WSS_HOST_PORT_PAIRS:
-      results[collider_instance] = self.probe_collider_instance(
-          collider_instance)
+    for instance in constants.WSS_INSTANCES:
```

Comment:
I don't understand your confusion. What do you suggest?

On Thu, Mar 5, 2015 at 2:33 PM, Donald Ephraim Curtis <
notifications@github.com> wrote:

> In src/app_engine/probers.py
> <https://github.com/webrtc/apprtc/pull/108#discussion_r25908743>:
>
> > @@ -155,23 +187,23 @@ def create_collider_active_host(self, old_active_host, probing_results):
> >
> >    def get(self):
> >      results = {}
> > -    for collider_instance in constants.WSS_HOST_PORT_PAIRS:
> > -      results[collider_instance] = self.probe_collider_instance(
> > -          collider_instance)
> > +    for instance in constants.WSS_INSTANCES:
>
> I get it now. It's very confusing since we're keying off host and one case
> we're keying off something else. I dunno.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/webrtc/apprtc/pull/108/files#r25908743>.
>



-- 
Jiayang Liu